build-info-extractor/src/main/java/org/jfrog/build/extractor/usageReport/UsageReporter.java

```diff
@@ -0,0 +1,51 @@
+package org.jfrog.build.extractor.usageReport;
+
+/**
+ * Created by Bar Belity on 20/11/2019.
+ */
+public class UsageReporter {
```

Comment:
I suggest renaming this class from *UsageReporter* to *UsageReport*, because it is merely a POJO. It doesn't actually perform the reporting (so it is not the reporter).